src/cotletico_manager.py

```diff
@@ -5,24 +5,6 @@
 import match
 
 
-def schedule_from_json(filename: str):
```

Comment:
The **_schedule_from_json_** method has been removed from _cotletico_manager.py_ but not moved to _schedule.py_.

Move to _schedule.py_ to schedule class and rename method to **_from_json_file_**.

Reply:
Added a method from the match_generator branch